pkg/nmi/server/redirector_windows.go

```diff
@@ -16,7 +16,7 @@ import (
 var podMap = make(map[types.UID]string)
 
 // WindowsRedirector returns sync function for windows redirector
-func WindowsRedirector(server *Server, subRoutinedone chan bool, mainRoutineDone chan bool) func(*Server, chan bool, chan bool) {
+func WindowsRedirector(server *Server) func(*Server, chan bool, chan bool) {
 	exit := make(chan struct{})
```

Comment:
This `exit` channel is completely private to the caller and is not connected to termination logic

Reply:
what about this?

Reply:
Updated the code.